app/controllers/concerns/my_facilities_filtering.rb

```diff
@@ -12,19 +12,22 @@ module MyFacilitiesFiltering
     before_action :populate_facility_sizes
     before_action :set_selected_facility_sizes
 
-    def filter_facilities
+    def filter_facilities #10
+      # debugger
       filtered_facilities = facilities_by_facility_group(@accessible_facilities)
       filtered_facilities = facilities_by_zone(filtered_facilities)
       facilities_by_size(filtered_facilities)
     end
 
     private
 
-    def populate_accessible_facilities
+    def populate_accessible_facilities #1
+      #  debugger
       @accessible_facilities = current_admin.accessible_facilities(:view_reports)
     end
 
-    def populate_facility_groups
+    def populate_facility_groups #2
+      #  debugger
```

Comment:
Can you remove all these commented out lines please? It will make things easier to scan and follow in terms of what is all changed here.